plugins/system/deborphan

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# Plugin to monitor the number of orphaned packages on the
+# system (using deborphan). Might work on section distib, who knows...
+#
+# Based on the debsecan plugin by Nicolas Bouthors. 2016-09-02
+#
+# Olivie Mehani <shtrom@ssji.net>
+#
+# Licence : GPLv2
+#
+#%# family=auto
+#%# capabilities=autoconf
+
+# Auto enable if we have deborphan only
+if [ "$1" = "autoconf" ] ; then
+  if [ -x /usr/bin/deborphan ]; then
+    echo yes
+  else
+    echo no
+  fi
+  exit 0
+fi
+
+# Fail if we don't have deborphan
+if [ ! -x /usr/bin/deborphan ]; then
+  exit 1
```

Comment:
Please emit a warning instead of failing silently. This helps users when trying "munin-run".